src/transformers/models/llama/tokenization_llama_fast.py

```diff
@@ -151,9 +151,6 @@ def __init__(
         self.legacy = legacy
 
         if add_prefix_space is not None:
-            logger.warning_once(
```

Comment:
Why remove this?

Reply:
It's not helpful and conversion time is fast, and if we save (serialize it) then you have the warning while you shuld not!